src/animals/Crocodile.ts

```diff
@@ -1,34 +1,43 @@
+import { NEGATIVE_LENGTH_ERROR_MESSAGE } from "../../tests/errorMessages";
 import { families } from "../app";
 import { AnimalCandidate } from "../hierarchy/Animal";
 import Reptile from "../hierarchy/Reptile";
-const NUMBER_OF_BABY_CROCODILES_BORN = 8;
-const NUMBER_OF_BABY_CROCODILES_BORN_RANDOM_GENDER = 5;
+export const NUMBER_OF_BABY_CROCODILES_BORN = 8;
+export const NUMBER_OF_BABY_CROCODILES_BORN_RANDOM_GENDER = 5;
 
 export default class Crocodile extends Reptile {
   private length: number;
   private numberOfBabiesBorn = NUMBER_OF_BABY_CROCODILES_BORN;
-  private numberOfBabiesBornRandomGender = NUMBER_OF_BABY_CROCODILES_BORN_RANDOM_GENDER;
+  private numberOfBabiesBornRandomGender =
+    NUMBER_OF_BABY_CROCODILES_BORN_RANDOM_GENDER;
 
   constructor(name: string, age: number, isMale: boolean, length: number) {
     super(name, age, isMale);
     this._canHaveFamily = true;
     if (length <= 0) {
-      throw new Error("Length must be greater than zero");
+      throw new Error(NEGATIVE_LENGTH_ERROR_MESSAGE);
     }
     this.length = length;
   }
 
   public giveBirth(): void {
-    const family = families.find(x => x.name == this.home);
-    const candidateCrocks: AnimalCandidate[] =
-      this.generateBabyProperties(this.numberOfBabiesBorn, this.numberOfBabiesBornRandomGender);
+    const family = families.find((crcodile) => crcodile.name == this.home);
+    const candidateCrocks: AnimalCandidate[] = this.generateBabyProperties(
+      this.numberOfBabiesBorn,
+      this.numberOfBabiesBornRandomGender
+    );
     let length: number;
     if (candidateCrocks) {
-      candidateCrocks.forEach(crockObjectInfo => {
+      candidateCrocks.forEach((crockObjectInfo) => {
```

Comment:
you and Kamen have different formatting settings and there are many formatting changes in this PR. I think the best thing to do is for you both to sync your settings and have one PR that only deals with the formatting. we can discuss that in the next planning